jobsdb/jobsdb.go

```diff
@@ -1412,11 +1414,16 @@ func computeInsertVals(before, after []string) ([]string, error) {
 				return calculatedVals, fmt.Errorf("computed index is invalid. before: %v, after: %v, calculatedVals: %v", before, after, calculatedVals)
 			}
 		}
-
+		var comparatorBool bool
+		if skipZeroAssertionForMultitenant {
+			comparatorBool = len(calculatedVals) == 2
```

Comment:
why are we ommitting the `(before[0] == "0" && len(calculatedVals) == 3)` condition?

Reply:
This would convert the `_0` tables to `_0_0_1` tables and other tables like `_1` would become `_1_1`. Just to be consistent irrespective of the index we can skip the length 3 condition for `multi-tenant` migrations and to ensure we are backwards compatible have the old flow too